drivers/sdcard_spi/sdcard_spi.c

```diff
@@ -199,22 +199,21 @@ static sd_init_fsm_state_t _init_sd_fsm_step(sdcard_spi_t *card, sd_init_fsm_sta
 
         case SD_INIT_SEND_ACMD41_HCS:
             DEBUG("SD_INIT_SEND_ACMD41_HCS\n");
-            int acmd41_hcs_retries = 0;
+            const uint32_t acmd41_hcs_retry_timeout = xtimer_now_usec() + (uint32_t)INIT_CMD_RETRY_US;
```

Comment:
```suggestion
            const uint32_t acmd41_hcs_retry_timeout = xtimer_now_usec() + INIT_CMD_RETRY_US;
```

Reply:
Unnecessary casts were removed in a fixup.